wiki/en/Include-Shared-Commands.md

```diff
@@ -7,6 +7,7 @@
 - `-p` or `--port`           Sets the local UDP port number. Default is 22124
 - `--jsonrpcport`            Enables JSON-RPC API server to control the app, set TCP port number (EXPERIMENTAL, APIs might change; only accessible from localhost). Please see [the JSON-RPC API Documentation file](https://github.com/jamulussoftware/jamulus/blob/master/docs/JSON-RPC.md).
 - `--jsonrpcsecretfile`      Required when using `--jsonrpcport`. Sets a path to a text file containing an authentication string for getting access to the JSON-RPC API.
+- `--jsonrpcbindip` 		 The IP address the JSON-RPC server should bind to. (optional, defaults to 127.0.0.1)
```

Comment:
The spacing looks off at least here on Github, probably as there's a tab while other places use spaces.

Reply:
None of this will render nicely. Seems like a markdown table is needed.

Reply:
We used to have that and it made it even harder to read :-)

Reply:
https://tucc.us/u/ouchy.html

Maybe we could roll our own HTML rather than using Markdown's design.

Reply:
[Is this harder to read?](https://github.com/jamulussoftware/jamuluswebsite/pull/877#issuecomment-1336112230).

Reply:
Not sure if it's wise to use in-line HTML (if that's what's happening - or it is working with the CSS?). Also it needs to work with https://jamulus.io/wiki/Software-Manual#command-line-options - that's why it's an include.

Reply:
Yes, this is in-line HTML with a pinch of CSS. I am curious what Markdown would make, and haven't looked yet. Markdown lets HTML co-exist inline (user beware, sure), and presumably we have our own CSS such that we can tweak the standard table if need be.

Reply:
Actually just tried it and the markdown doesn't look too bad. Just means the Client manual will have two tables but that doesn't look awful. It's easier to read. I'll give it a go.

Reply:
I'll wait for the PR to finalise before making the change though. Making a note on my PR so I don't forget.